### PR TITLE
MM-39165: Make "Go to channel" more prominent

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- Style the "Go to channel" button as primary
- Render it to the right of all other buttons
- The icon should always be the one in the ticket

@abhijit-singh The icons are the same size as before, but I was wondering if you would prefer them a bit smaller?
image

#### Ticket Link
- 

#### Checklist

- [ ] Telemetry updated
- [ ] Gated by experimental feature flag
- [ ] Unit tests updated

### CODE REVIEW

File: webapp/src/components/backstage/playbook_runs/shared.tsx [hunk 1]
```diff
@@ -70,13 +70,23 @@ export const SecondaryButtonLargerRight = styled(SecondaryButtonLarger)`
     margin-left: auto;
 `;
 
+export const ExpandRight = styled.div`
+    margin-left: auto;
+`;
```
Comment (agarciamontoro) [inline webapp/src/components/backstage/playbook_runs/shared.tsx:75]: Neat!

**Review by agarciamontoro — APPROVED** [review]
Thanks!

**Review by iomodo — APPROVED** [review]
Looks Great!

**Review by abhijit-singh — APPROVED** [review]
I agree @cpoile. They can be 16px instead of the current 18px. Everything else looks great!

File: webapp/src/components/backstage/playbook_runs/shared.tsx [hunk 2]
```diff
@@ -70,16 +70,32 @@ export const SecondaryButtonLargerRight = styled(SecondaryButtonLarger)`
...
     position: unset;
     height: unset;
     white-space: nowrap;
 `;
+
+export const Icon16 = styled.div`
+    > i {
+      font-size: 16px;
+    }
+`;
```
Comment (cpoile) [inline webapp/src/components/backstage/playbook_runs/shared.tsx:101]: @calebroseland I hate doing this, but directly styling the `i` element doesn't work (the `font-size` stays `18px`). Do you know why?

Comment (calebroseland) [reply webapp/src/components/backstage/playbook_runs/shared.tsx:101]: Selector specificity, I think. It's not that pretty, but if you need to arbitrarily increase it, there's:

```tsx
export const Icon16 = styled.i`
    && {
        font-size: 16px;
    }
`;
```

Comment (cpoile) [reply webapp/src/components/backstage/playbook_runs/shared.tsx]: Thank you, @calebroseland! That was what I was looking for.

**Pull request merged**